node/src/models/Lesson.ts

```diff
@@ -38,9 +38,13 @@ export interface Lesson extends Document {
   image: string;
   expectations: [LessonExpectation];
   conclusion: [string];
-  createdBy: string;
   lastTrainedAt: Date;
   features: any;
+  createdBy: mongoose.Types.ObjectId;
+  contentManagers: [mongoose.Types.ObjectId];
```

Comment:
assuming we change graphql so lesson has a `userPerms` query, then we won't need this (just flag `User.isContentManager` instead)